deploy/charts/hcloud-csi/README.md

```diff
@@ -0,0 +1,253 @@
+
+# Hcloud CSI
+
+This deploys a production-ready helm chart for the Hcloud CSI.
+
+## TL;DR
+
+```console
+helm repo add syself https://charts.syself.com
+helm install csi syself/hcloud-csi
+```
+
+## Introduction
+
+
+## Prerequisites
+
+- Kubernetes 1.19+
+- Helm 3.2.0+
+
+## Installing the Chart
+
+To install the chart with the release name `csi`:
+
+```console
+helm install csi syself/hcloud-csi
+```
+
+The command deploys hcloud-csi on the Kubernetes cluster in the default configuration. The [Parameters](#parameters) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `csi` deployment:
+
+```console
+helm delete csi
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Parameters
```

Comment:
For now, we avoided doing something like this for the hccm chart. Instead there's just a link to the current values.yaml.

That's just a personal preference of mine: I don't find the autogenerated READMEs of the values.yaml to be any more useful than just looking at the current version of the values.yaml :sweat_smile: 

I'm asking for it here mostly just to keep consistency with the hccm chart. If folks want some kind of autogenerated docs, I'd prefer we think about them a bit more generally first (and perhaps generate them from JSON Schema or something).